src/queues/job-queue.js

```diff
@@ -184,7 +184,7 @@ const initQueue = () => {
     })
   );
   // BetterSocialCronQueue.addCron(generalDailyQueue, '0 0,12,18 * * *', 'dailyRssUpdate');
-  BetterSocialCronQueue.addCron(generalDailyQueue, '0 12 * * *', 'dailyCredderUpdate');
+  // BetterSocialCronQueue.addCron(generalDailyQueue, '0 12 * * *', 'dailyCredderUpdate');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the cron job for `dailyCredderUpdate` has been commented out. If this is intentional and you want to disable Credder functionality, it's fine. However, if you plan to re-enable this in the future, consider using a feature flag or configuration option instead of commenting out the code. This would make it easier to enable or disable the feature without modifying the codebase.

```diff
- // BetterSocialCronQueue.addCron(generalDailyQueue, '0 12 * * *', 'dailyCredderUpdate');
+ const enableCredder = false; // Or get this value from a config file or environment variable
+ if (enableCredder) {
+   BetterSocialCronQueue.addCron(generalDailyQueue, '0 12 * * *', 'dailyCredderUpdate');
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->